entrypoint-monitoring/lambda/dist/lambda.js

```diff
@@ -10478,25 +10478,45 @@ var require_EndpointParameters = __commonJS({
 var require_package = __commonJS({
   "node_modules/@aws-sdk/client-cloudfront/package.json"(exports, module2) {
     module2.exports = {
-      name: "@aws-sdk/client-cloudfront",
-      description: "AWS SDK for JavaScript Cloudfront Client for Node.js, Browser and React Native",
-      version: "3.441.0",
-      scripts: {
-        build: "concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
-        "build:cjs": "tsc -p tsconfig.cjs.json",
-        "build:docs": "typedoc",
-        "build:es": "tsc -p tsconfig.es.json",
-        "build:include:deps": "lerna run --scope $npm_package_name --include-dependencies build",
-        "build:types": "tsc -p tsconfig.types.json",
-        "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
-        clean: "rimraf ./dist-* && rimraf *.tsbuildinfo",
-        "extract:docs": "api-extractor run --local",
-        "generate:client": "node ../../scripts/generate-clients/single-service --solo cloudfront"
+      _args: [
+        [
+          "@aws-sdk/client-cloudfront@3.441.0",
+          "/Users/justinbahr/Documents/LCM/Git_Code/mds-terraform-common/entrypoint-monitoring/lambda"
+        ]
+      ],
+      _from: "@aws-sdk/client-cloudfront@3.441.0",
+      _id: "@aws-sdk/client-cloudfront@3.441.0",
+      _inBundle: false,
+      _integrity: "sha512-cUWBim4YyOtQkFLUsw9LPfnnXJUPPy1IGGuJijBgd1OAaFTn3R45c4Jssegt/DACIRrirt67QDErphGkgJYkYQ==",
+      _location: "/@aws-sdk/client-cloudfront",
+      _phantomChildren: {},
+      _requested: {
+        type: "version",
+        registry: true,
+        raw: "@aws-sdk/client-cloudfront@3.441.0",
+        name: "@aws-sdk/client-cloudfront",
+        escapedName: "@aws-sdk%2fclient-cloudfront",
+        scope: "@aws-sdk",
+        rawSpec: "3.441.0",
+        saveSpec: null,
+        fetchSpec: "3.441.0"
+      },
+      _requiredBy: [
+        "/"
+      ],
+      _resolved: "https://registry.npmjs.org/@aws-sdk/client-cloudfront/-/client-cloudfront-3.441.0.tgz",
+      _spec: "3.441.0",
+      _where: "/Users/justinbahr/Documents/LCM/Git_Code/mds-terraform-common/entrypoint-monitoring/lambda",
```

Comment:
There's a lot of `/Users/justinbahr` in here, which doesn't seem right to me.

Reply:
resolved, not sure what happened with the build.